shared/settings/account/add-modals.tsx

```diff
@@ -17,7 +17,7 @@ export const Email = () => {
   const [email, onChangeEmail] = React.useState('')
   const [searchable, onChangeSearchable] = React.useState(true)
   const [addEmailInProgress, onAddEmailInProgress] = React.useState('')
-  const disabled = !email
+  const disabled = !email.trim()
```

Comment:
You should just trim this above and just refer to the trimmed  one

Reply:
Fair enough